tests/php/test-class-amp-meta-box.php

```diff
@@ -14,7 +16,7 @@
  *
  * @coversDefaultClass AMP_Post_Meta_Box
  */
-class Test_AMP_Post_Meta_Box extends WP_UnitTestCase {
+class Test_AMP_Post_Meta_Box extends DependencyInjectedTestCase {
```

Comment:
Is this still needed?

```suggestion
class Test_AMP_Post_Meta_Box extends WP_UnitTestCase {
```